lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +74,47 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        file = json.load(open(self._path_to_config))
+
+        config = ConfigDTO(
+            seed_urls=file['seed_urls'],
+            headers=file['headers'],
+            total_articles_to_find_and_parse=file['total_articles_to_find_and_parse'],
+            encoding =file['encoding'],
+            timeout=file['timeout'],
+            should_verify_certificate=file['should_verify_certificate'],
+            headless_mode=file['headless_mode']
+        )
+        return config
+
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        config = json.load(open(self._path_to_config))
+        if not isinstance(config['seed_urls'], list) or \
+                not all(re.match("https?://(www.)?", x) for x in config['seed_urls']):
+            raise IncorrectSeedURLError
+
+        num = config['total_articles']
+        if not (1 <= num <= 150):
+            raise NumberOfArticlesOutOfRangeError
+
+        if not isinstance(num, int):
+            raise IncorrectNumberOfArticlesError
+
+        if not isinstance(config['headers'], dict):
+            raise IncorrectHeadersError
+
+        if not isinstance(config['encoding'], str):
+            raise IncorrectEncodingError
+
+        if not isinstance(config['timeout'], int) or not (0 < config['timeout'] < 60):
```

Comment:
open conditions. we should avoid making `not (CONDITION and CONDITION)`, it is hard to read and understand

Reply:
fixed, I guess